### PR TITLE
add pt-br language support

### PR DESCRIPTION
**Pull request by ggfto**
Add support for pt-br language.

Have you personally loaded this code into an updated qbcore project and checked all it's functionality? YES
Does your code fit the style guidelines? YES
Does your PR fit the contribution guidelines? YES

### CODE REVIEW

File: locales/pt-br.lua [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        ["kick_all"] = "Expulsar todos os jogadores",
+        ["ammo_amount_set"] = "Defina sua quantidade de munição (somente administrador)",
+    }
+}
+
+if GetConvar('qb_locale', 'en') == 'pt-br' then
+    Lang = Locale:new({
+        phrases = Translations,
+        warnOnMissing = true,
+        fallbackLang = Lang,
```
Comment (GodyFromDiscord) [inline locales/pt-br.lua:278]: This line isn't needed just;

Lang = Locale:new({
    phrases = Translations,
    warnOnMissing = true
})

**Review by GodyFromDiscord — CHANGES_REQUESTED** [review]
There's a change I have requested, other than that the rest is okay.

**Comment by ChristianBDev** [comment]
@ggfto Please update the code to the requested changes so we can continue

**Comment by ggfto** [comment]
@ChristianBDev @GodyFromDiscord changes done folks!

**Pull request closed without merging**